### DISCUSSION TITLE
Help Me Get To 300?

### DISCUSSION THREAD

**Discussion by names12** [Jyers/Marlin#1118] [Q&A] [answered]
I know its been asked and answered a few times already. But the answers do not seem to make sense to me. Please be patient with me, I'm very new to firmware. 

I've been running Jyers E3V2-BLTouch-5x5-v4.2.2-v1.3.4 for a while now. Love it. No issues. But recently I've needed to start printing Polycarbonate, which requires hotter than stock firmware will allow. I've already purchased a new thermistor, heater, and have been running the MicroSwiss all metal hotend almost since day one. 

I see how I can change the settings in the configuration.h file, make the settings 15 degrees over, etc.

HOWEVER, I like running Jyers. Specifically the one already compiled for my printer that works, and works well. I've downloaded the "code" but have no idea what I need to do to get the same settings (the bltouch, 5x5, etc) and the temps set higher. 
It would be easy to edit the bin file, but I have no idea how, or even if its possible. 
I've googled the crap out of this, but all I seem to find is editing and compiling Marlin stock firmware. Seems like a good place to start, but is not specific enough for Jyers. PLEASE HELP ME

**Comment by chrisneal** [comment]
If you become a Jyers patreon (https://www.patreon.com/jyers), you get access to an automatic compilation server. You can upload your own configurations and it will build the firmware for you automatically 😊. 

Alternatively, you could follow this guide: https://automatedhome.party/2020/05/08/compiling-marlin-firmware-is-easy-dont-be-afraid-to-do-it/

**Comment by smooreace** [comment]
Jyers firmware IS Marlin. There is almost nothing different pertaining to
temperature modifications. The only thing I can think of that Jyers changes
over stock related to temps is a tweak to how thermal-runaway is
managed/mitigated.

There is no "editing .bin". You edit the (essentially) text files that tell
VSCode how to create the .bin file.

**Comment by names12** [comment]
I became a patreon! 
And I just uploaded a custom file. Hopefully it all works out. 

I get that jyers is marlin. I guess its hard for me to explain. I dont know what I need to know to ask the proper question. What I was trying to do was edit the configuration.h file with all the rest of the parameters and whatnot that I love about the jyers build that I have. Hopefully, with some luck, this build through the automatic compilation server will do the trick. I tried the simplified version. 
I'll be on discord later tonight.

**Comment by smooreace** [comment, marked as answer]
There is a pre-built set of config file that matches the pre-built firmware
that you were using. So all you really had to change was the hot end max
temp and the thermistor type

**Reply by names12** [reply]
I guess that's what I was looking for. I still cannot find them.

**Comment by smooreace** [comment]
In the.... Configuration Files... folder?

**Comment by smooreace** [comment]
I remembered the firmware you are using incorrectly. Thought you originally
said you were running the 3x3 firmware. So you would also have to change
the probe grid size to get back to 5x5

**Comment by names12** [comment]
OK... this solved my issue. Still gonna be a patreon and use the server and discord though. 
I just didnt know where to look. Like I said, I'm not a programmer. I know just enough to be incredibly dangerous to myself. 
Thank you for the help!
